entities.md

```diff
@@ -1,5 +1,5 @@
-Definitions of entities involved in the DAMS project.
+Definitions of entities involved in the Lock Keeper project.
 
 * Asset owner: the human that owns digital assets and the corresponding keys.
 * Key server: a server that participates in secure generation, storage, and use of a key (or key share).
-* User application: a middleman application that provides an interface between the asset owner and the DAMS local client API.
+* User application: an application that provides a direct user interface to the asset owner and calls the Lock Keeper local client API.
```

Comment:
I think we need to tweak this description and that this shouldn't be part of this PR. I recommend removing this text.